parsing/depend.ml

```diff
@@ -316,7 +316,9 @@ and add_modtype bv mty =
 
 and add_module_alias bv l =
   try
-    add_parent bv l;
+    (* If we are in delayed dependencies mode, we delay the dependencies
+       induced by "Lident s" *)
+    (if !Clflags.transparent_modules then add_parent else addmodule) bv l;
```

Comment:
The code makes it hard to know which of the two statements may raise `Not_found`; am I correct that only `lookup_map` may raise here? If so, could we rewrite the code to take the first statement out of the `try .. with` block?

Reply:
Indeed, fixed.